### DISCUSSION TITLE
Lowest value first

### DISCUSSION THREAD

**Discussion by Chaps79** [PlaceholderAPI/PlaceholderAPI#880] [(2) Support]
Hi, how can i get a placeholder to count the lowest value as the "top" place?  For instance, lowest time on a parkour should be first place.
Example using Parkour plugin:
ajlb_lb_parkour_player_personal_best_Easy_milliseconds_1_alltime_value%'
I want line to return the lowest time.  In my example, it returns the highest time."

I am trying to create a leaderboard for best times on a parkour course.  It should look like:
1. Name: Time
2. Name: Time
3. ect.

**Comment by Andre601** [comment]
I think it would be best to ask the dev of AJLeaderboards this question as their plugin handles what value is displayed at what position.

**Reply by Chaps79** [reply]
ok thanks
